### PR TITLE
WIP: jsonschema: use external package

### PR DESCRIPTION
**Pull request by david-caro**
* Use the external package as the source for the jsonschemas instead
  of the internal ones.
WIP: requires inspire-schemas to be already published on pypi

Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ services:
       - ACCOUNTS_SESSION_REDIS_URL=redis://redis:6379/2
       - SEARCH_ELASTIC_HOSTS=indexer
       - ES_BULK_TIMEOUT=240
+      - CELERY_RDB_HOST=0.0.0.0
```
Comment (david-caro) [inline docker-compose.yml]: I should remove this...

File: docker-compose.yml [hunk 2]
```diff
@@ -57,6 +58,8 @@ services:
     command: celery worker -E -A inspirehep.celery --loglevel=INFO --purge
     volumes_from:
       - static
+    ports:
+      - "6000:7000"
```
Comment (david-caro) [inline docker-compose.yml]: and this

File: inspirehep/modules/literaturesuggest/tasks.py [hunk 3]
```diff
@@ -58,6 +58,7 @@ def formdata_to_model(obj, formdata):
     # ======
     if '$schema' in data and not data['$schema'].startswith('http'):
         jsonschemas_ext = current_app.extensions.get('invenio-jsonschemas')
+
```
Comment (kaplun) [inline inspirehep/modules/literaturesuggest/tasks.py]: and this :)

File: setup.py [hunk 4]
```diff
@@ -220,7 +220,7 @@
             'inspirehep_holding_js = inspirehep.modules.workflows.bundles:details_js',
         ],
         'invenio_jsonschemas.schemas': [
-            'inspire_records = inspirehep.modules.records.jsonschemas',
+            'inspire_records = inspire_schemas',
```
Comment (kaplun) [inline setup.py]: Ooh: So it was just that simple?

Comment (david-caro) [inline setup.py]: Yep, it was easy :smiley:, just needed the proper packaging on the inspire_schemas side.

File: inspirehep/modules/literaturesuggest/tasks.py (hunk 3 above)
Comment (david-caro) [inline inspirehep/modules/literaturesuggest/tasks.py]: yep, that was where the 'rdb.set_trace()' was :clap:

**Comment by jacquerie** [comment]
http://www.folklore.org/StoryView.py?story=Negative_2000_Lines_Of_Code.txt

**Comment by david-caro** [comment]
Ha! it seems it was counting the json code for the coverage total...

**Pull request closed without merging**